File: 0861-flipping-an-image/0861-flipping-an-image.java
class Solution {
    public int[][] flipAndInvertImage(int[][] image) {
        int row = image.length;
        int col = image[0].length;

        for(int i = 0; i < row; i++){
            for(int j = 0; j < col / 2; j++){
                int temp = image[i][j];
                image[i][j] = image[i][col-j-1];
                image[i][col-j-1] = temp;
            }
        }
        for(int i = 0; i < row; i++){
            for(int j = 0; j < col; j++){
                if(image[i][j] == 1) image[i][j] = 0;
                else image[i][j] = 1;
            }
        }
        return image;
    }
}

File: 0861-flipping-an-image/0861-flipping-an-image.py
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        row = len(image)
        col = len(image[0])

        for i in range(row):
            # for j in range(col // 2):
            #     image[i][j], image[i][col-j-1] = image[i][col-j-1], image[i][j]
            image[i] = image[i][::-1]
        
        for i in range(row):
            for j in range(col):
                if image[i][j] == 1:
                    image[i][j] = 0
                else:
                    image[i][j] = 1
        
        return image